src/test-e2e/po/form.po.ts

```diff
@@ -86,13 +101,16 @@ export class FormComponent extends Component {
     return this.getField(ctrlName).getAttribute('aria-describedby').then(id => this.locator.element(by.id(id)).getText());
   }
 
+  getText = (ctrlName: string, isTextInput: boolean): promise.Promise<string> =>
```

Comment:
Would be nice if we could get the tagName and work this out ourselves, rather than having to pass the boolean.